pkg/webui/components/navigation/side/item/item.styl

```
@@ -118,3 +128,41 @@
 
   &-open
     transform: translateY(-50%) rotate(180deg)
+
+.fly-out-list
+  visibility hidden
```

```suggestion
  display: none
```

Note also that you forgot the `:` here.